core/src/scss/utilities/mixins/link/_external-link.scss

```diff
@@ -13,17 +13,22 @@
 @mixin external-link($external-link: external-link, $external-link-hover: external-link-hover) {
   &::after {
     @include margin(null null -1px 4px);
-    background: url("#{$image-path}/#{$external-link}.png") no-repeat 0 0;
-    background: url("#{$image-path}/#{$external-link}.svg") no-repeat 0 0;
-    background-size: 100%;
-    content: '';
+    @include size(0.65em);
     display: inline-block;
-    height: 0.65em;
-    width: 0.65em;
+    content: '';
+    mask: url("#{$image-path}/#{$external-link}.svg") no-repeat 0 0;
+    mask-size: cover;
+    background-color: $color-bright-red;
   }
 
   &:hover::after {
-    background-image: url("#{$image-path}/#{$external-link-hover}.png");
-    background-image: url("#{$image-path}/#{$external-link-hover}.svg");
+    background-color: $color-black;
+  }
+
+  // IE only - hides icon.
+  @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
Can we make this a mixin? This seems like something we would want to standardize and have available.

Reply:
Sounds like a great idea :)